Source/Inspector/InspectorWindow.cpp

```diff
@@ -114,11 +113,14 @@ void InspectorWindow::ShowMeshRenderComponent(Scene& scene)
 				for(int i=0;i<size;i++)
 					filePath+=*(payloadPointer+i);
 				std::filesystem::path payloadPath = std::filesystem::path(filePath);
+
 				if(payloadPath.extension()==".obj")
 				{
-					std::cout<<"NAME: "<<payloadPath.filename().string()<<"\n";
-					//Maybe we SetMesh on _activeGameObject
-					//_activeGameObject->GetComponent<RenderComponent>()->SetMesh(mesh);
+					//Set The mesh that we dragged to the RenderComponent
+					auto asset_Helper = _assetHelper->GetMeshAsset(payloadPath.string());
+					std::string f = (std::get<1>(asset_Helper)->fileName);
```

Comment:
Here the second value in the tuple is the texture. This could technically be nullptr and so this line would cause a crash. The tuple should first check if 0 is true (that a mesh was found) and then it can safely use the second part of the tuple. Fortunately it looks like f isn't used and this line should be deleted anyway